robots/cmd/flakefinder/README.md

```diff
@@ -21,6 +21,55 @@ Selecting the right builds:
 - Only shows test results for all lanes where a test at least failed once on one of the found lanes
 - Only take prow jobs into account which were run on the commit which got merged
 
+Reports entry page
+------------------
+
+The [flakefinder reports entry page](https://storage.googleapis.com/kubevirt-prow/reports/flakefinder/index.html) shows an overview over all projects for which flakefinder reports are generated.
+
+How to read flakefinder reports
+-------------------------------
+
+Once you select a project, you are taken to the report overview page.
+
+### project report overview page
+
+![Flakefinder reports entrypage for project](ff_overview.png)
+
+1. daily rolling window report (updated hourly), covers all PRs merged today
+1. daily report for yesterday, covers all PRs merged yesterday
+1. weekly report from yesterday, covers all PRs merged from six days before yesterday until yesterday
+
+_Not shown here: flakefinder reports for last four weeks (updated once a week)_
+
+### project report details page
+
+In a nutshell the daily report of day x covers all merged PRs from the start of the day before until the runtime of the report.
+
+Each report contains rows for all tests where at least one test failed on any lane for all PRs that were merged at or after the report date. Rows are sorted according to severity of test failures (i.e the more test fails, the more the test appears at the top of the report
+
+Columns are:
+1. link index
+2. test name
+3. lane aggregation for the test under scope \
+   the numbers are: \
+   red: no of fails \
+   green: number of passes \
+   gray: no of skips
```

Comment:
for consistency - it might be better to use "number" instead of the abbreviation